.travis.yml

```diff
@@ -4,6 +4,9 @@
 language: objective-c
 
 env:
+  matrix:
+    
+    - CONDA_PERL=5.20.3.1
```

Comment:
@bekozi we removed `perl` from the docker image, so now we need to re-add it everywhere it is needed. Here `perl` is used in some of the tests scripts.